src/io/flutter/view/FlutterView.java

```diff
@@ -451,32 +455,67 @@ protected void handleJxBrowserInstalled(FlutterApp app, InspectorService inspect
   private void presentDevTools(FlutterApp app, InspectorService inspectorService, ToolWindow toolWindow, boolean isEmbedded) {
     assert(SwingUtilities.isEventDispatchThread());
 
-    presentLabel(toolWindow, INSTALLING_DEVTOOLS_LABEL);
+    devToolsInstallCount += 1;
+    presentLabel(toolWindow, getInstallingDevtoolsLabel());
 
     openInspectorWithDevTools(app, inspectorService, toolWindow, isEmbedded);
+
+    setUpToolWindowListener(app, inspectorService, toolWindow, isEmbedded);
   }
 
+  @VisibleForTesting
+  protected void setUpToolWindowListener(FlutterApp app, InspectorService inspectorService, ToolWindow toolWindow, boolean isEmbedded) {
+    if (this.toolWindowListener == null) {
+      this.toolWindowListener = new FlutterViewToolWindowManagerListener(myProject);
+    }
+    this.toolWindowListener.updateOnWindowOpen(() -> {
+      devToolsInstallCount += 1;
+      presentLabel(toolWindow, getInstallingDevtoolsLabel());
+      openInspectorWithDevTools(app, inspectorService, toolWindow, isEmbedded, true);
+    });
+  }
+
+  private String getInstallingDevtoolsLabel() {
+    return "<html><body style=\"text-align: center;\">Installing DevTools... <br><br>If this takes more than 15 seconds, you can restart " +
```

Comment:
Move this to FlutterBundle.properties. We only need the English text there so you can leave the HTML here if you want. Line 36 has an example of using a macro.